src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/DIY/Skellige/Silver/KnutTheCallous.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Alsein.Extensions;
+using System;
+
+namespace Cynthia.Card
+{
+    [CardEffectId("70080")]//冷酷无情的克努特 KnuttheCallous
+    public class KnuttheCallous : CardEffect
+    {//Deploy: discard a card from your hand, then generate a bronze unit from your starting deck, strengthen it by 1 and add it to your hand
+        public KnuttheCallous(GameCard card) : base(card) { }
+        public override async Task<int> CardPlayEffect(bool isSpying, bool isReveal)
+        {
+            //discard a card from your hand
+            var cards = await Game.GetSelectMenuCards(PlayerIndex, Game.PlayersHandCard[PlayerIndex]);
+            if (!cards.TrySingle(out var target))
+            {
+                return 0;
+            }
+            var targetGroup = target.Status.Group;
+            if (!Game.PlayerBaseDeck[PlayerIndex].Deck.Where(x => x.Group == targetGroup).Select(x => x.CardId).TryMessOne(out var targetId, Game.RNG))
+            {
+                return 0;
+            }
```

Comment:
Hmm, what is the purpose of this code? It looks like it is checking whether the starting deck contains cards with the same Group (Gold/Silver/Bronze) as the selected card?

Reply:
Not totally sure of the purpose, I took it from the Vigo code, the card works fine on my end so I'd keep that.